frontend/packages/kubevirt-plugin/src/components/modals/nic-modal/nic-modal.tsx

```diff
@@ -96,15 +96,22 @@ export const Network: React.FC<NetworkProps> = ({
     allowPodNetwork,
     allowedMultusNetworkTypes,
   ).filter((n) => n.getType().isSupported());
+  const validationMessage = nadsLoadError
+    ? 'Error fatching available Network Attachment Definitions. Contact your system admin for additional support'
```

Comment:
I think we are saying administrator right?
```suggestion
    ? 'Error fetching available Network Attachment Definitions. Contact your system administrator for additional support'
```

Reply:
+1 Thanks :exclamation: